torch/_torch_docs.py

```diff
@@ -3969,6 +3969,30 @@ def merge_dicts(*dicts):
 
 """)
 
+add_docstr(torch.polygamma,
+           r"""
+polygamma(n, input, out=None) -> Tensor
+
+Computes the :math:`n^{th}` derivative of the digamma function on :attr:`input`, where :attr:`n \geq 0`
```

Comment:
This sentence reads a little weird to me because the two clauses convey different information. How about the following:

Computes the :math:`n^{th}` derivative of the digamma function on :attr:`input`. :attr:`n \geq 0` is called the *order* of the polygamma function.

Reply:
Thanks @zou3519 for the suggestion. Not sure what I was thinking there.